selftests/unit/test_utils_partition.py

```diff
@@ -33,7 +33,7 @@ def cannot_sudo(command):
     try:
         process.run(command, sudo=True)
         False
-    except process.CmdError:
+    except (process.CmdError, OSError):
```

Comment:
The proper fix would be to make `process` to report `CmdError` in such case. It's a minor thing so let me treat it in a separate PR (to avoid unnecessary delay)